src/docs/product/dashboards/widget-library/index.mdx

```diff
@@ -15,13 +15,15 @@ The widget library contains a collection of prebuilt widgets you can add to your
 The library includes the following widgets:
 
 - **Duration Distribution**: A multi-overlay chart visualizing the distribution of transaction duration across various percentiles (p50, p75, and p95)
-- **High Throughput Transactions**: The top five transactions with the largest number of events
+- **High Throughput Transactions**: The top transactions with the largest number of events
 - **LCP by Country**: World map showing the p75 of page load times for each country
 - **Miserable Users**: The total number of unique users who have experienced slow transactions (transaction duration greater than 1200ms)
 - **Slow vs. Fast Transactions**: Bar chart comparing the percentage of transactions that are over 300ms (slow) and under 300ms (fast)
 - **Issues For Review**: A table of unresolved issues for review, ordered by the most recently seen issues
-- **Top Unhandled Error Types**: The top five most frequently encountered unhandled errors
+- **Top Unhandled Error Types**: The top most frequently encountered unhandled errors
 - **Users Affected by Errors**: A comparison of the total number of errors and the number of unique users affected by the errors
+- **Crash Rates for Recent Releases**: Percentage of crashed sessions for your recent releases
+- **Session Health**: The total number of abnormal, crashed, errored and healthy sessions
```

Comment:
these two widget are only available for EA, should there be an indication of that?

Reply:
Addressed in this commit https://github.com/getsentry/sentry-docs/pull/5083/commits/9f2e2ce97d8c0235582dbc665bc16691d5c26d36#diff-124b2f7356d01f77a1ac84d799606e8247298ade1ab46fdae30038d4afbef784R28-R33